packages/composite-checkout/src/lib/localize.js

```diff
@@ -0,0 +1,25 @@
+/* @format */
+
+/**
+ * External dependencies
+ */
+import { useContext } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CheckoutContext from './checkout-context';
+
+// TODO: we need to implement this; probably delegated to i18n-calypso
```

Comment:
Would using wp-i18n work instead? That can be externalised for core since it ships in WordPress so saves the bundle size in wp-admin.

Reply:
That's a good idea. I'll take a look when we get to this part.